src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/osd/osd-scrub-modal/osd-scrub-modal.component.html

```diff
@@ -8,10 +8,8 @@
           [formGroup]="scrubForm"
           novalidate>
       <div class="modal-body">
-        <div *ngIf="selected.length === 1">
-          <p i18n>You are about to apply a {deep, select, 1 {deep }}scrub to
-            the OSD <strong>{{ selected[0].id }}</strong>.</p>
-        </div>
+        <p i18n>You are about to apply a {deep, select, 1 {deep }}scrub to
+          the OSD(s): <strong>{{ selected.toString() }}</strong>.</p>
```

Comment:
I vote for implementing a pipe as suggested above because there are multiple code places where you could use it.